creme/sketch/static/sketch/js/chart/barchart.js

```diff
@@ -92,15 +99,17 @@ creme.D3BarChart = creme.D3Chart.sub({
         var bounds = creme.svgBounds(sketch.size(), props.margin);
 
         var ymax = d3.max(data, function(d) { return d.y; }) || 1;
-        var yAxisTitleHeight = creme.d3FontSize(chart.select('.y.axis')) * 2;
+        var yAxisFontSize = creme.d3FontSize(chart.select('.y.axis'));
+        var yAxisTitleHeight = yAxisFontSize * 2;
+        var yAxisSize = Math.max(props.yAxisSize, 6 + (yAxisFontSize * 3));
```

Comment:
Hum maybe some comment about the formula :thinking:

Reply:
"green powder" does not work ? :sweat_smile: